pkgs/development/python-modules/img2pdf/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, isPyPy, python3 }:
+
+with python3.pkgs;
+
+buildPythonPackage rec {
+  pname = "img2pdf";
+  version = "0.2.4";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1bn9zzsmg4k41wmx1ldmj8a4yfj807p8r0a757lm9yrv7bx702ql";
+  };
+
+  propagatedBuildInputs = [
+    pillow
+    pdfrw
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = https://gitlab.mister-muffin.de/josch/img2pdf;
+    description = "Losslessly convert raster images to PDF. ";
+    license = lib.licenses.lgpl3;
```

Comment:
Since you are using `with stdenv.lib;` lib is already scoped and this could be shortened to `license = licenses.lgpl3;`.